etl-server.js

```diff
@@ -127,5 +124,21 @@ server.register([
     server.start(function() {
       server.log('info', 'Server running at: ' + server.info.uri);
     });
+    server.on('response', function (request) {
+      if (request.response === undefined || request.response.statusCode != 200){
+        console.log("there is a problem");
```

Comment:
We need to account for cases where a user was denied access i.e 403 status. In that regard, the above if clause is limiting. Get rid of it and just log everything regardless of the status.